src/components/transaction/Details.vue

```diff
@@ -357,6 +419,29 @@ export default class TransactionDetails extends Vue {
       this.timelockStatus = this.$t("TRANSACTION.TIMELOCK.UNKNOWN");
     }
   }
+
+  public getMilestone(config, height?: number): { [key: string]: any } {
+    if (!height && this.height) {
+        height = this.height;
+    }
+
+    if (!height) {
+        height = 1;
+    }
+    let milestoneIdx = 0;
+    while (
+        milestoneIdx < config.milestones.length - 1 &&
+        height >= config.milestones[milestoneIdx + 1].height
+    ) {
+        milestoneIdx++;
+    }
+
+    while (milestoneIdx && height < config.milestones[milestoneIdx].height) {
+        milestoneIdx--;
+    }
```

Comment:
Not very readable :smile: Comment?

Reply:
agree... its code from ark-crypto but i don't see the point....